source/administration/changelog.md

```diff
@@ -2,6 +2,168 @@
+    - Added `"EnableDiagnostics": true` to enable sending error reporting and diagnostic information to Mattermost to improve the quality and performance of the software.
+
+**Additional Changes to Enterprise Edition:**    
+
+The following config settings will only work on servers with an Enterprise License that has the feature enabled.
+ 
+- Under `LdapSettings` in `config.json`:
+    - `"FirstNameAttribute": ""` is no longer a required field
+    - `"LastNameAttribute": ""` is no longer a required field
+- Under `SamlSettings` in `config.json`:
+    - `"FirstNameAttribute": ""` is no longer a required field
+    - `"LastNameAttribute": ""` is no longer a required field
+
+### Database Changes from v3.3 to v3.4
+
+**Status Table**
+
+ - Added `Manual` column.
+ - Added `ActiveChannel` column.
+
+### API Changes from v3.3 to v3.4
+
+**New routes:**
+ - Added `GET` at `/oauth/authorized`
+     - Returns the OAuth2 Apps authorized by the user.  On success it returns a list of sanitized OAuth2 Authorized Apps by the user.
+ - Added `POST` at `/oauth/"+clientId+"/deauthorize`
+     - Deauthorizes a user on an OAuth 2.0 app, where `clientId` corresponds to the application. Returns status OK on success or an AppError on fail. 
+ - Added `POST` at `/oauth/"+clientId+"/regen_secret`
+     - Generates a new OAuth App Client Secret, where `clientId` corresponds to the application. Returns an OAuth2 App on success. Must be authenticated as a user and the same user who registered the app or a System Admin. 
+ - Added `POST` at `/admin/ldap_test`
+     - Will run a connection test on the current LDAP settings. It will return the standard OK response if settings work. Otherwise it will return an appropriate error.
+ - Added `POST` at `/users/status/set_active_channel`
+     - Sets the Status.ActiveChannel field which is used to tell if the user is actively viewing a channel or not.
+ - Added `GET` at `/admin/recently_active_users/{teamId}`
+     - Returns a list of recent active users. 
+
+### Known Issues
+
+- Deleted messages don't disappear from the channel for the user who deleted the message, if the message was previously edited and right hand sidebar is open.
+- A single collapsed link or image preview re-opens after refresh.
+- Files sent in private chat to members in a different team are not accessible.
+- YouTube videos show as “Video not found” on Desktop App.
+- “More” option under Direct Message list no longer shows count of team members not in your direct message list.
+- On Firefox, CTRL/CMD + U keyboard shortcut doesn’t work.
+- Webhook attachments don’t show up in search results.
+- Messages sometimes don't appear deleted until the page is refreshed.
+- When joining a channel from a public link, the page sometimes loads for a long time and requires a refresh.
+- Recent mentions search for usernames containing a dot returns results containing only part of the username, due to database search limitations.
```

Comment:
Since this is a database limitation, does this need to be included? I think maybe it should just live in the search known issues.